### PR TITLE
Enhancement/fix sheet model multiple timing + sheet highlighting + multiple fixes (See commits)

### PR DESCRIPTION
**Pull request by Erik9519**
#### What have I done
Fixed sheet positioning for multiple timing files loaded.
Added ability to highlight sheets through sheet selector.
Fixed viewport keys working when they are not supposed to (like when typing in sheet selector)

#### Testing
Load files and observe the sheet moving and ability to highlight sheets

#### Addresses SRD section(s)
SheetModel (added setHighlight function, minor change to constructor)

### CODE REVIEW

File: src/app/core/models/sheet-model.ts [hunk 1]
```diff
@@ -538,4 +563,13 @@ export class SheetModel {
 
         return texture;
     }
+
+    setHighlight(val: boolean) {
```
Comment (Dacaspex) [inline src/app/core/models/sheet-model.ts]: Doc comment?

File: src/app/workspace/viewport/viewport.component.ts [hunk 2]
```diff
@@ -295,11 +300,22 @@ export class ViewportComponent implements AfterViewInit, OnDestroy {
...
                                 sheetTimingState.getLastPOIName(),
                                 this.printer.getRouteCurve(),
-                                SheetCurveAlignment.CENTER, // default value
+                                curveAlignment,
                                 0, // default value
                                 this.route.getSegmentPosition(sheetTimingState.getPosition()).areSheetsUpsideDown()
                             );
                         }
+                        // Alternate alignment between Top and Bottom if initial alignment was center
+                        if (curveAlignment === SheetCurveAlignment.TOP) {
```
Comment (Dacaspex) [inline src/app/workspace/viewport/viewport.component.ts]: The flipping of the sheet happens somewhat implicitly, perhaps we can make a constant that defines for each timing file index a position?
```
const sheetAlignmentsMap = [
    SheetCurveAlignment.TOP, 
    SheetCurveAlignment.Bottom
]
...
let index = 1;
for (...) {
    const sheetCurveAlignment = this.sheetAlignmentsMap[index];
    index++;
}
```

File: src/app/core/models/sheet-model.ts (hunk 1 above)
Comment (Erik9519) [reply src/app/core/models/sheet-model.ts]: Added

File: src/app/workspace/viewport/viewport.component.ts (hunk 2 above)
Comment (Erik9519) [reply src/app/workspace/viewport/viewport.component.ts]: Addressed

**Comment by Dacaspex** [comment]
Please keep the 
> One issue, one branch, one commit
workflow. Right now I have to review a couple of issues, which makes it harder

File: src/app/workspace/viewport/viewport.component.ts [hunk 3]
```diff
@@ -84,6 +89,11 @@ export class ViewportComponent implements AfterViewInit, OnDestroy {
      */
     private readonly GRID_COLOR = 0x333333;
 
+    /**
+     * Printer boundaries for moving
+     */
+    private readonly MODEL_BOUNDARIES = 50;
```
Comment (Dacaspex) [inline src/app/workspace/viewport/viewport.component.ts]: Should be 500

**Pull request merged**